docs/api/conformance/Metadata-Tests.md

```diff
@@ -16,54 +16,54 @@ You will need to pre-populate the AppDirectory with the following items:
+| Test | 4. Confirm1        | Compare the `AppMetadata` object to the expected definition for the fields provided above during setup and ensure that the metadata matches.  |
+| Test | 5. getAppMetadata2 | Retrieve metadata for the second instance of the app with <br/>`const metadata2 = fdc3.getAppMetadata(appIdentifier2)`  |
+| Test | 6. Confirm2        | An `instanceId` should be provided, confirm that it matches the one in `appIdentifier2`  |
 
 - `AppInstanceMetadata`: Perform the above steps.
 
 ## Finding Instances
 
-| App | Step           | Details                                                                                           |
-|-----|----------------|---------------------------------------------------------------------------------------------------|
-| Test   | 1.Open1    | Open the first instance of App A using <br /> `const appIdentifier1 = await fdc3.open({appId: "<A's appId>"})` <br />and confirm that its `AppIdentifier` contains an `instanceId`.  |
-| Test   | 2.Open2    |Open a second instance of App A using <br />`const appIdentifier2 = await fdc3.open({appId: "<A's appId>"})` <br />and confirm that its `AppIdentifier` contains an `instanceId` and that its value differs from that returned for the first instance. |
-| Test   | 3.FindInstances    | Retrieve details of open instances of app A with <br /> `let instances = await fdc3.findInstances({appId: "<A's appId>"})` <br /> confirm that both `appIdentifier1` and `appIdentifier2` are both present in the array.  |
-| Test   | 4.RaiseIntent   | Use `appIdentifier1` to raise an intent and target that instance, with<br /> `const resolution = fdc3.raiseIntent("aTestingIntent", {"type": "testContextX"}, appIdentifier1)` |
-| Test   | 5.Confirm1 | Check that `resolution.source` matches `appIdentifier1` |
-| A | 6.ConfirmReceipt | Ensure that the instance of app A represented by `appIdentifier1` received the raised intent |
+| App  | Step              | Details                                                                                           |
+|------|-------------------|---------------------------------------------------------------------------------------------------|
+| Test | 1. Open1          | Open the first instance of App A using <br/> `const appIdentifier1 = await fdc3.open({appId:"<A's-appId>"})` <br/>and confirm that its `AppIdentifier` contains an `instanceId`.  |
+| Test | 2. Open2          | Open a second instance of App A using <br />`const appIdentifier2 = await fdc3.open({appId:"<A's-appId>"})` <br/>and confirm that its `AppIdentifier` contains an `instanceId` and that its value differs from that returned for the first instance. |
+| Test | 3. FindInstances  | Retrieve details of open instances of app A with <br/> `let instances = await fdc3.findInstances({appId:"<A's-appId>"})` <br/> confirm that both `appIdentifier1` and `appIdentifier2` are both present in the array. |
```

Comment:
Is `-` needed in `({appId:"<A's-appId>"})`?

Reply:
Possibly doen to prevent the a line break?